torchci/lib/fetchCommit.ts

```diff
@@ -1,18 +1,26 @@
 import _ from "lodash";
 import { Octokit } from "octokit";
 import rocksetVersions from "rockset/prodVersions.json";
-import { queryClickhouseSaved } from "./clickhouse";
+import { enableClickhouse, queryClickhouseSaved } from "./clickhouse";
 import { commitDataFromResponse, getOctokit } from "./github";
 import { removeCancelledJobAfterRetry } from "./jobUtils";
 import getRocksetClient from "./rockset";
 import { CommitData, JobData } from "./types";
 
 async function fetchDatabaseInfo(owner: string, repo: string, sha: string) {
-  if (false) {
-    const response = queryClickhouseSaved("commit_jobs_query", {
+  if (enableClickhouse()) {
+    const response = await queryClickhouseSaved("commit_jobs_query", {
       repo: `${owner}/${repo}`,
       sha: sha,
     });
+
+    for (const row of response) {
```

Comment:
I'm curious about the reason for this custom transformation. I guess this is to make the output of ClickHouse match what Rockset returns.  However, it makes more sense to perform these steps inside the SQL logic instead, i.e. `toInt64(durationS)` instead of `parseInt(row.durationS)` here

Reply:
The type of durationS is an int64 on the clickhouse UI, idk why it gets converted to string by the API, so I manually convert it back

Reply:
Does CH include a type or something in it's response? I'm wondering if this converter can be generalized

Reply:
afaict some response formats do while others don't.  I think it also depends on language/library used.  I don't think .json() does and I'm not super sure if javascript has this format at all